sdk/src/common/gql.ts

```diff
@@ -118,21 +152,26 @@ function getQueryBody(args: QueryBodyGQLArgsType): string {
 		? JSON.stringify(args.tags)
 				.replace(/"(name)":/g, '$1:')
 				.replace(/"(values)":/g, '$1:')
+				.replace(/"match"/g, 'match')
 				.replace(/"FUZZY_OR"/g, 'FUZZY_OR')
 		: null;
 	const owners = args.owners ? JSON.stringify(args.owners) : null;
+	const recipients = args.recipients ? JSON.stringify(args.recipients) : null;
 	const cursor = args.cursor && args.cursor !== CURSORS.end ? `"${args.cursor}"` : null;
 
 	let fetchCount: string = `first: ${paginator}`;
 	let txCount: string = '';
 	let nodeFields: string = `data { size type } owner { address } block { height timestamp }`;
 	let order: string = '';
+	let recipientsfield: string = ''
 
-	switch (args.gateway) {
+	const gateway = args.gateway ?? GATEWAYS.goldsky;
+	switch (gateway) {
 		case GATEWAYS.arweave:
 			break;
 		case GATEWAYS.goldsky:
-			txCount = `count`;
+			if (!cursor) txCount = `count`;
+			if (recipients) recipientsfield = `recipients: ${recipients}`;
```

Comment:
arweave.net supports the `recipients` filter on the `transactions` query, so shouldn't this line also be added to the other `case`?